src/Tizen.NUI/src/public/XamlBinding/Page.cs

```diff
@@ -211,6 +211,31 @@ public bool SendBackButtonPressed()
             return OnBackButtonPressed();
         }
 
+        /// <summary>
+        /// Equality operator
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static bool operator ==(Page x, Page y)
```

Comment:
== is same as ReferenceEquals.
why overload it???

Reply:
Because Page inherit BaseHandle and BaseHandle overload ==/!=,  it will compare from native handle in ==/!= . We overload ==/!= to avoid throw exception when user check if page is null.